code/game/objects/items.dm

```diff
@@ -652,6 +652,83 @@
 	if(SSmapping.configs[GROUND_MAP].environment_traits[MAP_COLD] && (flags_item_map_variant & ITEM_ICE_PROTECTION))
 		min_cold_protection_temperature = ICE_PLANET_MIN_COLD_PROTECTION_TEMPERATURE
 
+/**
+	Play small animation and jiggle when picking up an object
+ */
+
+/obj/item/proc/do_pickup_animation(atom/target)
+	set waitfor = FALSE
```

Comment:
note to self

Reply:
I actually don't know what's being requested here.

Reply:
Nothing